src/pages/loading-waiting-states/spinner-overlays/data.tsx

```diff
@@ -0,0 +1,6 @@
+    eulaConfirmation: 'I have read and agree to the Terms & Conditions',
```

Comment:
Feels a little weird to have this stored in an object here...I would probably just hardcoded this value into the label prop for the checkbox as a string and leave this file as just the eula text.